test/format/format_config.c

```diff
@@ -707,9 +707,22 @@ config_cache(void)
         if (config_explicit(NULL, "cache.minimum") && GV(CACHE) < GV(CACHE_MINIMUM))
             testutil_die(EINVAL, "minimum cache set larger than cache (%" PRIu32 " > %" PRIu32 ")",
               GV(CACHE_MINIMUM), GV(CACHE));
+        if (config_explicit(NULL, "cache.maximum") && GV(CACHE) > GV(CACHE_MAXIMUM))
+            testutil_die(EINVAL, "cache set larger than maximum cache (%" PRIu32 " > %" PRIu32 ")",
+              GV(CACHE), GV(CACHE_MAXIMUM));
         goto dirty_eviction_config;
     }
 
+    /*
+     * Check if both min and max cache sizes have been specified, that minimum is set less than
+     * maximum.
+     */
+    if (config_explicit(NULL, "cache.minimum") && config_explicit(NULL, "cache.maximum") &&
+      GV(CACHE_MINIMUM) > GV(CACHE_MAXIMUM))
```

Comment:
Either the comment doesn't match the code, or this is an accidental bug. The minimum is permitted to equal the maximum here. (FWIW I think it's reasonable to allow equality and force a specific size, but then the comment should be updated to "less than or equal".)

Reply:
Oops! sorry fixed it here [03f0045](https://github.com/wiredtiger/wiredtiger/pull/10806/commits/03f004549589bb673ae066d52bf2916eb34f9630)

Reply:
Did you link the wrong commit? That change seems to be the `/proc/meminfo` fix.

Reply:
Never mind - I can see the comment is fixed now :-)